doc/Preboarding_and_Onboarding.rst

```diff
@@ -51,6 +51,14 @@ The preboarding process is the first process that must be completed after config
 Onboarding
 ===========
 
+If the Elasticsearch service is not running, execute the below mentioned commands.
+
```

Comment:
Its a known reported Elasticsearch issue which can be avoided by executing mentioned commands

Reply:
Resolved.